apps/docs/pages/guides/platform/network-restrictions.mdx

```diff
@@ -8,9 +8,7 @@ export const meta = {
 
 <Admonition type="note">
 
-Network Restrictions are currently in beta and are being slowly made available to all projects.
-
-The current iteration of Network Restrictions applies to connections to Postgres and PgBouncer. Support for restricting connections to PostgREST, Storage, and Auth will be added at a later date.
+Network Restrictions is currently in beta and is slowly being made available to all projects. [Contact support](https://app.supabase.com/support/new) if you'd like to request early access.
```

Comment:
The callout was getting a bit unwieldy so I moved a sentence to Getting started and the other to Limitations.

Reply:
thanks!